core/src/main/java/com/yandex/yoctodb/v1/immutable/V1DatabaseReader.java

```diff
@@ -33,6 +31,24 @@
 @ThreadSafe
 public class V1DatabaseReader extends DatabaseReader {
     private static final int DIGEST_BUF_SIZE = 4096;
+    private static final Set<Integer> SUPPORTED_FORMATS;
+
+    static {
+        Set<Integer> supported = new HashSet<>();
+        supported.add(6);
+        supported.add(V1DatabaseFormat.FORMAT);
+
+        SUPPORTED_FORMATS = Collections.unmodifiableSet(supported);
+    }
+
+    public static String supportedFormatsString() {
+        StringBuilder sb = new StringBuilder();
+        for (Integer format : SUPPORTED_FORMATS) {
+            sb.append("<").append(format).append("> ");
+        }
+
+        return sb.toString();
```

Comment:
You can write a `Stream`-based one-liner if we are going to switch to Java 8.

Reply:
Dropped jdk7, moved to streams, thanks!